instrumentation/elasticsearch/elasticsearch-rest-common/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/elasticsearch/rest/ElasticsearchRestNetResponseAttributesGetter.java

```diff
@@ -21,28 +21,13 @@ public String transport(ElasticsearchRestRequest request, Response response) {
 
   @Override
   @Nullable
-  public String peerName(ElasticsearchRestRequest request, @Nullable Response response) {
-    if (response != null) {
-      return response.getHost().getHostName();
-    }
-    return null;
```

Comment:
The `elasticsearch-rest` instrumentations are the only ones that actually lost some data because of this change - but I'd argue that it's not that important, because it uses the Apache HTTP client to make all the REST calls, so you'll still have the peer name&port on the child HTTP CLIENT span.

Reply:
I'll open an issue to track this, but agree it's not that important